doc/classes/HTTPRequest.xml

```diff
@@ -295,6 +296,7 @@
 			Request exceeded its maximum size limit, see [member body_size_limit].
 		</constant>
 		<constant name="RESULT_BODY_DECOMPRESS_FAILED" value="8" enum="Result">
+			Request failed due to an error while decompressing the response body. Possible causes include unsupported or incorrect compression format, corrupted data, or incomplete transfer.
```

Comment:
```suggestion
			The request failed due to an error while decompressing the response body. Possible causes include unsupported or incorrect compression format, corrupted data, or incomplete transfer.
```

Reply:
Thank you for the suggestion! However, I believe 'Request failed...' fits better with the style used throughout the documentation and helps maintain consistency.

Reply:
Oh yeah makes sense